code/src/App.js

```diff
@@ -1,9 +1,85 @@
-import React from 'react'
+import React, { useEffect, useState } from "react";
+
+import { API_URL, LIKE_URL } from "./utils/urls";
+import ThoughtsItem from "./components/ThoughtsItem";
+import ThoughtsForm from "./components/ThoughtsForm";
+import LoadingItem from "./components/LoadingItem";
 
 export const App = () => {
+  const [thoughts, setThoughts] = useState([]);
+  const [newThought, setNewThought] = useState("");
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(null);
```

Comment:
If you did not use error state variable anyway, you can just delete it from your code, because your button is anyway disabled to send any error type of message :)